### PR TITLE
Irminger 9 recovered

### PR DESCRIPTION
**Pull request by cdobs**

### CODE REVIEW

**Comment by rtravis-whoi** [comment]
Not sure why but I can't submit my comments using the green review button and Approve the PR - tried Friday and this morning.  PR is Approved.

File: GI01SUMO/GI01SUMO_R00009_ingest.csv [hunk 1]
```diff
@@ -0,0 +1,86 @@
...
+mi.dataset.driver.wavss_a.dcl.wavss_a_dcl_recovered_driver,/omc_data/whoi/OMC/GI01SUMO/R00009/cg_data/dcl12/wavss/*.wavss.log,GI01SUMO-SBD12-05-WAVSSA000,recovered_host,Available,
+mi.dataset.driver.metbk_a.dcl.metbk_a_dcl_recovered_driver,/omc_data/whoi/OMC/GI01SUMO/R00009/cg_data/dcl12/metbk2/*.metbk2.log,GI01SUMO-SBD12-06-METBKA000,recovered_host,Available,
+mi.dataset.driver.metbk_ct.dcl.metbk_ct_dcl_driver,/omc_data/whoi/OMC/GI01SUMO/R00009/cg_data/dcl12/metbk2/*.metbk2.log,GI01SUMO-SBD12-06-METBKA001,recovered_host,Available,
+mi.dataset.driver.metbk_ct.metbk_ct_recovered_driver,/omc_data/whoi/OMC/GI01SUMO/R00009/instruments/dcl12/METBK-CT*/*.hex,GI01SUMO-SBD12-06-METBKA001,recovered_inst,Available,
+mi.dataset.driver.fdchp_a.dcl.fdchp_a_dcl_recovered_driver,/omc_data/whoi/OMC/GI01SUMO/R00009/cg_data/dcl12/fdchp/*.fdchp.log,GI01SUMO-SBD12-08-FDCHPA000,recovered_host,Available,
+mi.dataset.driver.fdchp_a.fdchp_a_recovered_driver,/omc_data/whoi/OMC/GI01SUMO/R00009/instruments/dcl12/FDCHP_sn*/D*/fdchp_*.dat,GI01SUMO-SBD12-08-FDCHPA000,recovered_inst,Available,
```
Comment (cooleyky) [inline GI01SUMO/GI01SUMO_R00009_ingest.csv:86]: Looks good except for CPM file name mask

**Review by cooleyky — CHANGES_REQUESTED** [review]
The CPM file name mask is pointing to an irid2shore directory that doesn't exist yet in the raw data repo. There is a cpm_status.txt file, but it's in the syslog directory instead: https://rawdata-west.oceanobservatories.org/files/GI01SUMO/R00009/cg_data/cpm1/
Everything else in these ingestion CSVs looks good.

File: GI01SUMO/GI01SUMO_R00009_ingest.csv [hunk 2]
```diff
@@ -0,0 +1,86 @@
...
+mi.dataset.driver.pco2w_abc.pco2w_abc_recovered_driver,/omc_data/whoi/OMC/GI01SUMO/R00009/instruments/dcl11/imm/PCO2W_snC0140/SAMI*.txt,GI01SUMO-RII11-02-PCO2WC053,recovered_inst,Available,
+mi.dataset.driver.phsen_abcdef.imodem.phsen_abcdef_imodem_recovered_driver,/omc_data/whoi/OMC/GI01SUMO/R00009/cg_data/dcl11/imm/phsen1/*.DAT,GI01SUMO-RII11-02-PHSENE041,recovered_host,Available,
+mi.dataset.driver.phsen_abcdef.phsen_abcdef_recovered_driver,/omc_data/whoi/OMC/GI01SUMO/R00009/instruments/dcl11/imm/PHSEN_snP0172/SAMI*.txt,GI01SUMO-RII11-02-PHSENE041,recovered_inst,Available,
+mi.dataset.driver.phsen_abcdef.imodem.phsen_abcdef_imodem_recovered_driver,/omc_data/whoi/OMC/GI01SUMO/R00009/cg_data/dcl11/imm/phsen2/*.DAT,GI01SUMO-RII11-02-PHSENE042,recovered_host,Available,
+mi.dataset.driver.phsen_abcdef.phsen_abcdef_recovered_driver,/omc_data/whoi/OMC/GI01SUMO/R00009/instruments/dcl11/imm/PHSEN_snP0177/SAMI*.txt,GI01SUMO-RII11-02-PHSENE042,recovered_inst,Available,
+mi.dataset.driver.cg_cpm_eng.cpm.cg_cpm_eng_cpm_recovered_driver,/omc_data/whoi/OMC/GI01SUMO/R00009/cg_data/cpm1/irid2shore/cpm_status*.txt,GI01SUMO-SBC11-00-CPMENG000,recovered_host,Available,
```
[the remark this answers is not in the document]
Comment (cooleyky) [reply GI01SUMO/GI01SUMO_R00009_ingest.csv:65]: @cdobs confirmed that the directory exists, and this CSV is fine as is

**Review by cooleyky — APPROVED** [review]
Looks good after getting confirmation that filename masks are good

**Pull request merged**